README.md

```diff
@@ -30,40 +30,46 @@ Example of events:
 58m   Normal  CREATE                   ConfigMap   ConfigMap default/ingress02-nginx-ingress-controller
 ```
 
-### Installation
+## Installation
 
 Use the [Helm](https://helm.sh/) chart from this repo:
 
-```
+```sh
 helm install chart/
 ```
 
 Or use the chart from [deliveryhero/helm-charts/stable/k8s-event-logger](https://github.com/deliveryhero/helm-charts/tree/master/stable/k8s-event-logger):
 
-```
+```sh
 helm repo add deliveryhero https://charts.deliveryhero.io/
 helm install deliveryhero/k8s-event-logger
 ```
 
-Or use the docker image [maxrocketinternet/k8s-event-logger](https://hub.docker.com/r/maxrocketinternet/k8s-event-logger)
+Or use the pre-built image [maxrocketinternet/k8s-event-logger][pre-built image]
 
-#### Building a container image
+### Building a container image
 
-If you're unable to use the [prebuilt][image] docker image, you can build it yourself:
+If you're unable to use the [pre-built image], you can build it yourself:
 
 ```sh
-make IMG=maxrocketinternet/k8s-event-logger TAG=latest
+make all IMG=<your-container-registry>/k8s-event-logger TAG=latest
```

Comment:
This does the the full multi-arch setup, too. (the "all" target, that is). I found it easier to test with that and then a `make clean` afterwards, but feel free to reject this for the simpler make image.